ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh

```diff
@@ -122,7 +122,7 @@ install_rhtap(){
     apiVersion: tekton.dev/v1
     kind: PipelineRun
     metadata:
-      generateName: redhat-trusted-application-pipeline-pe-info-
+      generateName: rhtap-pe-info-
```

Comment:
should  not be trusted-*

Reply:
@Roming22  the value of `{{ .Chart.Name }}` is `trusted-application-pipeline` , see https://github.com/redhat-appstudio/rhtap-installer/blob/main/Chart.yaml#L13C7-L13C24. so `generateName: trusted-application-pipeline-pe-info-`

Reply:
@flacatus The pipeline name is changing again, cf https://github.com/redhat-appstudio/rhtap-installer/pull/103

The very long name was causing issues.